specification/paths/Clients-client_id.json

```diff
@@ -0,0 +1,118 @@
+{
+  "get": {
+    "tags": [
+      "Clients"
+    ],
+    "security": [
+      {
+        "OAuth2": [
+          "organizations.manage"
```

Comment:
I don't think we need a scope to create a client at the moment, do we?

Reply:
Well, at the moment you can't really create a client. The task describes we should add crud permissions for clients and add them to the organizations.manage scope, so that's what I did.

Reply:
I see, fair enough then.